index.html

```diff
@@ -57,25 +57,235 @@ <h2>you!</h2>
-                <img src="img/Vector(1).png" alt="ector1">
-            </a>
-            <a href="https://github.com/Danieldotcomcoder" target="_blank" rel="noreferrer noopener">
-                <img src="img/Vector(2).png" alt="vector2">
-            </a>
-            <a href="#" target="_blank" rel="noreferrer noopener">
-                <img src="img/Vector(3).png" alt="vector3">
-            </a>
+        <ul class="social-media-icons">
+            <li> <a href="https://twitter.com/DannyDotcoder" target="_blank" rel="noreferrer noopener">
+                    <img src="img/Vector.png" alt="vector">
+                </a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/daniel-shdeed-832b03115/" target="_blank"
+                    rel="noreferrer noopener">
+                    <img src="img/Linkedinicon.png" alt="linkedinicon">
+                </a>
+            </li>
+            <li>
+                <a href="#" target="_blank" rel="noreferrer noopener">
+                    <img src="img/Vector(1).png" alt="ector1">
+                </a>
+            </li>
+            <li>
+                <a href="https://github.com/Danieldotcomcoder" target="_blank" rel="noreferrer noopener">
+                    <img src="img/Vector(2).png" alt="vector2">
+                </a>
+            </li>
+            <li>
+                <a href="#" target="_blank" rel="noreferrer noopener">
+                    <img src="img/Vector(3).png" alt="vector3">
+                </a>
+            </li>
+        </ul>
+    </section>
+    <section class="works">
+        <div class="card-works">
+            <div class="snapshot-portfolio">
+                <img src="img/SnapshootPortfolio.png" alt="snapshootportfolio">
+            </div>
+            <div class="info-block">
+                <div class="primary-text-box">
+                    <div class="project-title">
+                        <h1>Tonic</h1>
+                    </div>
+                    <div class="project-frame">
+                        <h3>CANOPY</h3>
+                        <img src="img/Counter.png" alt="counter" width="8" height="8">
+                        <h4>Back End Dev</h4>
+                        <img src="img/Counter1.png" alt="counter1" width="8" height="8">
+                        <h4>2015</h4>
+                    </div>
+                </div>
+                <div class="project-description">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="project-tags">
+                    <img src="img/html.png" alt="html">
+                    <img src="img/css.png" alt="css">
+                    <img src="img/javascript.png" alt="javascript">
+                </div>
+                <div class="seeproject">
+                    <a href="#" rel="noreferrer noopener">See Project</a>
```

Comment:
Please add hover functionality to the `See project` button for all the projects.